File: DayliStudent_BackEnd/daylistudent/models/note.py
class Note(object):
    @staticmethod
    def toJson(id, group_id, username, title, body):
        note_schema = {
            'id': id,
            'groupId': group_id,
            'username': username,
            'title': title,
            'body': body
        }
        return note_schema

File: DailyStudent/src/main/java/pt/uc/cm/daylistudent/models/Note.kt
package pt.uc.cm.daylistudent.models

import java.io.Serializable

class Note(var id: String = "-1",
           var groupId: String = "-1",
           var username: String? = "",
           var title: String,
           var body: String) : Serializable {
    companion object {
        private const val serialVersionUID = 7526471155622776147L
    }
}


